### DISCUSSION TITLE
二维码——打印到终端还是直接显示图片好呢

### DISCUSSION THREAD

**Discussion by VermiIIi0n** [VermiIIi0n/fuckZHS#21] [Polls]
没什么用的投票~ 主要是为了试试这投票功能😄, 可能会根据结果改改默认配置...吧?

* 印图片的好处是显示出来小, 不会占终端显示空间, 也不会显示不全.

* 打印到终端好处是不用多一步点击关闭图片窗口, 而且在服务器上也能用.

**Comment by code-yuan-shi** [comment]
大佬呀，为什么我运行之后出了一堆  ?[0;37;47m

**Reply by code-yuan-shi** [reply]
另一个地方作者有说，我这里再说一次吧，在config设置ensure_unicode 为true或者show_in_terminal为false就行了
